### PR TITLE
COL-759 - Engagement Index box gets score, rank and formatting

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/COL-759

The box is wired.

### CODE REVIEW

File: public/app/dashboard/profile.css [hunk 1]
```diff
@@ -43,12 +43,27 @@
...
-  margin: 5px;
   padding: 10px 10px 10px 10px;
 }
 
+.profile-engagement-column h2 {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 1.3;
+  word-wrap: break-word;
+}
```
Comment (johncrossman) [inline public/app/dashboard/profile.css]: Despite decreased font-size, I keep `h2` for appropriate structure (accessibility).

File: public/app/dashboard/profile.html [hunk 2]
```diff
@@ -19,17 +19,23 @@ <h2 data-ng-bind="user.canvas_full_name"></h2>
...
+          <a tool-href data-tool="engagementindex" data-course="me.course">
+            Engagement Index
+          </a>
+        </h2>
+        <hr/>
+        <div class="profile-engagement-score" data-ng-class="{'profile-engagement-score-not-shared': me.id === user.id && !user.share_points}">
+          {{user.points}} {{user.points === 1 ? 'Point' : 'Points'}}
+        </div>
+        <div data-ng-if="userRank">
+          <span data-ng-bind='userRank'></span> out of <span data-ng-bind='courseUserCount'></span>
```
Comment (paulkerschen) [inline public/app/dashboard/profile.html]: @johncrossman Double quotes above?

**Comment by paulkerschen** [comment]
Nice ordinal function, @johncrossman! Will merge modulo Travis.

File: public/app/dashboard/profile.html (hunk 2 above)
Comment (johncrossman) [inline public/app/dashboard/profile.html]: @paulkerschen Whoopsie. Amended.

**Pull request merged**